Developer Credential Package Initial Phase (#1111)
Esri/arcgis-rest-js: demos/dev-credentials-client/README.md

# Client Web Demo for Developer Credential Package

This page will show API key and OAuth 2.0 app CRUD demo with help of `arcgis-rest-developer-credentials` package. First
step requires a user to log in
via [Authenticate with an ArcGIS identity](https://developers.arcgis.com/arcgis-rest-js/authentication/tutorials/authenticate-with-an-arcgis-identity-rest-js-browser/).

> Next, you will use the form create an API key, which will be stored in RAM for simplicity purpose.

Reviewer: ```suggestion
Then you can use the form to create an API key or switch to the OAuth 2.0 form to create an OAuth 2.0 app.
```

> To retrieve or update, please click a specific key in the table and continue your operation. If the named token is

Reviewer: Once you've created an API key or OAuth 2.0 app, you can click on the item in the table below the form to edit it.

expired, page will be auto-logout.

## Running this demo

1. Run `npm run build` in the repository's root directory.

> 2. Run `npm start` to spin up the development server.

Reviewer: ```suggestion
2. Go into `demos/dev-credentials-client` and run `npm start` to spin up the development server.
```

3. Visit [http://localhost:8080](http://localhost:8080).

> ## Credits

Reviewer: ```suggestion
## Additional tools used
```
